src/api/libcellml/component.h

```diff
@@ -122,6 +122,13 @@ class LIBCELLML_EXPORT Component: public ComponentEntity, public ImportedEntity
      */
     void removeMath();
 
+    /**
+     * @brief Return the id attribute from the math string, if present.
+     *
+     * @return id The @c std::string if the id attribute is present, or an empty string if not.
+     */
+    std::string mathId() const;
```

Comment:
This should be removed as it is not clear which ID would be returned.

Reply:
d4b98796